### ISSUE TITLE
Accessibility issue

### ISSUE THREAD

**Issue by RafaelFernandesBR** [UnigramDev/Unigram#1212]
Hello, unigram still has some issues with accessibility with screen readers.
For example, when we click on a person's profile, and click on the common groups button, he or she displays the groups like this:

chat {
  id = -1001166473003
  type = chatTypeSupergroup {
    supergroup_id = 1166473003
    is_channel = false
  }
  title = "Android For The Blind"
  photo = chatPhoto {
    small = file {
      id = 17
      size = 10393
      expected_size = 10393
      local = localFile {
        path = "C: \ Users \ technique \ AppData \ Local \ Packages \ 38833FF26BA1D.UnigramPreview_g9c9v27vpyspw \ LocalState \ 0 \ profile_photos \ 107736488_42413.jpg"
        can_be_downloaded = true
        can_be_deleted = true
        is_downloading_active = false
        is_downloading_completed = true
        download_offset = 0
        downloaded_prefix_size = 10393
        downloaded_size = 10393
      }
      remote = remoteFile {
        id = "AQADAQATqO1rBgAEwrLThpHHJgWtpQACAg"
        is_uploading_active = false
        is_uploading_completed = true
        uploaded_size = 10393
      }
    }
    big = file {
      id = 18
      size = 0
      expected_size = 0
      local = localFile {
        path = ""
        can_be_downloaded = true
        can_be_deleted = false
        is_downloading_active = false
        is_downloading_completed = false
        download_offset = 0
        downloaded_prefix_size = 0
        downloaded_size = 0
      }
      remote = remoteFile {
        id = "AQADAQATqO1rBgAERZ9q2jRI8x6vpQACAg"
        is_uploading_active = false
        is_uploading_completed = true
        uploaded_size = 0
      }
    }
  }
  last_message = message {
    id = 802160640
    sender_user_id = 887647312
    chat_id = -1001166473003
    sending_state = null
    is_outgoing = false
    can_be_edited = false
    can_be_forwarded = true
    can_be_deleted_only_for_self = false
    can_be_deleted_for_all_users = true
    is_channel_post = false
    contains_unread_mention = false
    date = 1570917385
    edit_date = 0
    forward_info = null
    reply_to_message_id = 0
    ttl = 0
    ttl_expires_in = 0.000000
    via_bot_user_id = 0
    author_signature = ""
    views = 0
    media_album_id = 0
    content = messageVoiceNote {
      voice_note = voiceNote {
        duration = 32
        waveform = bytes [63] {00 00 00 40 00 01 00 10 FE D7 20 04 62 52 B3 2C 04 73 52 A2 61 08 5A A9 0E 2B 00 C3 24 4D 84 AC 64 A3 2A B0 88 10 08 91 18 0F 67 94 72 FA 9F 10 0E 74 7B 82 A1 42 68 48 04 00 00 43 C2 3C 05}
        mime_type = "audio / ogg"
        voice = file {
          id = 841
          size = 74243
          expected_size = 74243
          local = localFile {
            path = "C: \ Users \ technique \ AppData \ Local \ Packages \ 38833FF26BA1D.UnigramPreview_g9c9v27vpyspw \ LocalState \ 0 \ voice \ 4976864590567571515.oga"
            can_be_downloaded = true
            can_be_deleted = true
            is_downloading_active = false
            is_downloading_completed = true
            download_offset = 0
            downloaded_prefix_size = 74243
            downloaded_size = 74243
          }
          remote = remoteFile {
            id = "AwADAQADOwAD-l8RRcnCg5N-soHdAg"
            is_uploading_active = false
            is_uploading_completed = true
            uploaded_size = 74243
          }
        }
      }
      caption = formattedText {
        text = ""
        entities = vector [0] {
        }
      }
      is_listened = true
    }
    reply_markup = null
  }
  order = 6747038793292841725
  is_pinned = false
  is_marked_as_unread = false
  is_sponsored = false
  can_be_deleted_only_for_self = false
  can_be_deleted_for_all_users = false
  can_be_reported = false
  default_disable_notification = false
  unread_count = 0
  last_read_inbox_message_id = 802160640
  last_read_outbox_message_id = 801112064
  unread_mention_count = 0
  notification_settings = chatNotificationSettings {
    use_default_mute_for = true
    mute_for = 0
    use_default_sound = true
    sound = "default"
    use_default_show_preview = true
    show_preview = false
    use_default_disable_pinned_message_notifications = true
    disable_pinned_message_notifications = false
    use_default_disable_mention_notifications = true
    disable_mention_notifications = false
  }
  pinned_message_id = 416284672
  reply_markup_message_id = 0
  draft_message = null
  client_data = ""
}
  1 of 1

I think I should just show the name of the groups.
When we do a search, it shows the results the same way.

In a group we are a manager of, we can't do things like remove a person or make him a manager.
The solution to this case, I think it would be, is to put the management options on the keyboard applications key.
That's it, sorry for my english!

**Comment by Yeldece** [comment]
Also can't read profile>files, links, voice messages etc. it appears:
message {
  id = 265005563904
  sender_user_id = 800544257
  chat_id = -1001060755082
  sending_state = null
  is_outgoing = false
 ... and more.

**Comment by FrayxRulez** [comment]
I fixed common groups with this commit: https://github.com/UnigramDev/Unigram/commit/68adf963f8972d18a99870044df78a3be668fcb2

**Comment by FrayxRulez** [comment]
We can probably use this ticket to collect all the lists that contain random text rather than useful info.

**Comment by FrayxRulez** [comment]
Shared media as well: https://github.com/UnigramDev/Unigram/commit/e19ecd1b0cab47d53771af575e033be09397a119

**Comment by masonasons** [comment]
Also, if you perform a search, search results show as: Unigram.Collections.KeyedList`2[System.String,System.Object]

**Comment by sukiletxe** [comment]
And below, some json about the message's information.

**Comment by FrayxRulez** [comment]
Fixed

**Comment by sukiletxe** [comment]
Now the only remaining things are forwards (as logged in #1446 ), the items in the mentions autocompletes, and bot commands.

**Comment by sukiletxe** [comment]
I think this should be reopened (see last message)
